### PR TITLE
add view plugin to SIMAApplicationInput.json

### LINKED ISSUE #1318: ViewPlugin for SimaAppInput

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## What does this pull request change?
add view plugin for SIMAApplicationInput available for all roles.
The design is almost the same as for the edit plugin.

image

## Issues related to this change
closes #1318

### CODE REVIEW

**Review by soofstad — APPROVED** [review]
Looks good.
But would it be possible to reuse the same component for the two different plugins?
Where one passes a "view/readOnly" flag, that toggles the components?
Or simply hides the "save" button.
The goal being to reduce the amount of components we need to maintain

**Review by netr0m — APPROVED** [review]
Looks good!

File: web/packages/plugins/ui/sima-application-input/src/EditSimaApplicationInput.tsx [hunk 1]
```diff
@@ -23,17 +26,47 @@ import {
...
   Row,
 } from './components'
 import _ from 'lodash'
 
+const ReadOnlyPathTextField = (props: { path: string }) => {
+  return (
+    <Tooltip
+      title={truncatePathString(props.path) === props.path ? '' : props.path}
+    >
+      <Input
```
Comment (eoaksnes) [inline web/packages/plugins/ui/sima-application-input/src/EditSimaApplicationInput.tsx:42]: Input fields have readOnly property. Maybe also use that to disable the possibility to write/change anything in the fields.

File: web/packages/plugins/ui/sima-application-input/src/EditSimaApplicationInput.tsx [hunk 2]
```diff
@@ -23,17 +26,47 @@ import {
...
+        type="string"
+        value={truncatePathString(props.path)}
+      />
+    </Tooltip>
+  )
+}
+
+const ReadOnlyTextField = (props: { text: string; label: string }) => {
+  return (
+    <TextField
```
Comment (eoaksnes) [inline web/packages/plugins/ui/sima-application-input/src/EditSimaApplicationInput.tsx:57]: TextFields have readOnly property. Maybe also use that to disable the possibility to write in the fields.

**Review by eoaksnes — APPROVED** [review]

**Pull request merged**